packages/matter.js/src/behavior/definitions/general-commissioning/ServerNodeFailsafeContext.ts

```diff
@@ -10,19 +10,14 @@ import { Endpoint } from "../../../endpoint/Endpoint.js";
 import { Fabric } from "../../../fabric/Fabric.js";
 import { Node } from "../../../node/Node.js";
 import { NetworkCommissioningBehavior } from "../network-commissioning/NetworkCommissioningBehavior.js";
-import { OperationalCredentialsBehavior } from "../operational-credentials/OperationalCredentialsBehavior.js";
 
 /**
  * {@link FailsafeContext} for {@link Node} API.
  */
 export class ServerNodeFailsafeContext extends FailsafeContext {
     #node: Node;
-    #operationalCredentialsRestored = false;
     #storedState?: {
         networks: Map<Endpoint, NetworkCommissioningBehavior.State["networks"]>;
-        nocs: OperationalCredentialsBehavior.State["nocs"];
```

Comment:
I feel like we're losing some code here that works with "native" matter semantics.  I know it's kind of redundant right now but if we move to using the native attributes as backing for Fabric API then we'll have to recreate this.  Just commenting it out could lead to bitrot but it might be nice to have as reference in the future.